content-security-policy/support/echo-embedding-csp.py

```diff
@@ -0,0 +1,11 @@
+import json
+def main(request, response):
+    header = request.headers.get("Embedding-CSP");
+    message = {}
+    message['embedding_csp'] = header if header else None
+    return [("Content-Type", "text/html")], '''
```

Comment:
I tried added `("Access-Control-Allow-Origin", "*")`
but that didn't seem to help with the commented out code in the html file?